staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -115,6 +115,8 @@ func (obj functionFakeRuntimeObject) DeepCopyObject() runtime.Object {
 // won't ever see that event, and will always see any event after they are
 // added.
 func (m *Broadcaster) blockQueue(f func()) {
```

Comment:
I think that this is the problem, we should rethink this to avoid this blockQueue "hack"

https://go.dev/blog/pipelines

Reply:
Agreed I'm looking at how we can get rid of this